client/src/components/Alert.tsx

```diff
@@ -17,17 +12,24 @@ const defaultMessages = {
 };
 
 const statusColor = {
-  error: 'bg-red-100 border-red-500 text-red-700',
-  success: 'bg-green-100 border-green-500 text-green-700',
-  info: 'bg-blue-100 border-blue-500 text-blue-700',
-  warning: 'bg-yellow-100 border-yellow-500 text-yellow-700',
+  error:
+    'bg-gray-100 dark:bg-neutral-700 border-gray-500 dark:text-neutral-200 text-gray-500',
+  success:
+    'bg-gray-100 dark:bg-neutral-700 border-green-500 dark:text-neutral-200 text-gray-500',
+  info: 'bg-gray-100 dark:bg-neutral-700 border-blue-500 dark:text-neutral-200 text-gray-500',
+  warning:
+    'bg-gray-100 dark:bg-neutral-700 border-yellow-500 dark:text-neutral-200 text-gray-500',
 };
 
 const statusIcon = {
-  error: <VscError className='text-red-700' size={20} />,
-  success: <AiOutlineCheckCircle className='text-green-700' size={20} />,
-  info: <AiOutlineInfoCircle className='text-blue-700 ' size={20} />,
-  warning: <IoWarningOutline className='text-yellow-700' size={20} />,
+  error: <IoWarning className='text-red-400 dark:opacity-80' size={25} />,
+  success: (
+    <AiFillCheckCircle className='text-green-400 dark:opacity-80' size={23} />
+  ),
+  info: (
+    <AiFillInfoCircle className='text-blue-300 dark:opacity-80' size={23} />
+  ),
+  warning: <IoWarning className='text-yellow-400 dark:opacity-80' size={25} />,
```

Comment:
I think we should have a different icon for `error` and `warning`. I suggest keeping the icon for warning but change the `error` icon to `VscError`